workloads/container-disks/virtio-win.md

```diff
@@ -0,0 +1,50 @@
+# Virtio win container disk
+
+**Provides**:
+
+* virtio drivers
+* QEMU guest agent
+
+Kubevirt distributes virtio package for windows in form if container disk.
+The package contains the virtio drivers as well as qemu guest agent.
+
+Package is intended to be used as CDROM attached to virtual machine with windows.
+It can be used either during install phase as SATA CDROM to provider drivers or
+in existing installation as SATA CDROM to provide QEMU guest agent installation.
+
+Example how to attach:
+
+```yaml
+apiVersion: kubevirt.io/v1alpha2
+kind: VirtualMachineInstance
+metadata:
+  creationTimestamp: null
+  labels:
+    special: vmi-virtio
+  name: vmi-virtio
+spec:
+  domain:
+    devices:
+      disks:
+      - disk:
+          bus: virtio
+        name: pvcdisk
+        volumename: pvcvolume
+      - cdrom:
+          bus: sata
+        name: containerdisk
+        volumeName: registryvolume
+    machine:
+      type: ""
+    resources:
+      requests:
+        memory: 64M
+  terminationGracePeriodSeconds: 0
+  volumes:
+  - persistentVolumeClaim:
+      claimName: disk-windows
+    name: pvcvolume
+  - containerDisk:
+      image: kubevirt/virtio-container-disk
```

Comment:
This actually raises that we probably want to rename the disk: What about `windows-driver-disk` @petrkotas ?

Reply:
Makes sense. Lets do it in separate PR to keep this simple. It would require renaming the disk itself.